index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Eve's Portfolio</title>
+    <script src="https://use.fontawesome.com/13c71f6777.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="vendor/styles/normalize.css">
+    <link rel="stylesheet" href="styles/icon.css">
+    <link rel="stylesheet" href="styles/base.css">
+    <link rel="stylesheet" href="styles/layout.css">
+    <link rel="stylesheet" href="styles/modules.css">
+  </head>
+  <body>
+
+    <nav class="main-nav-bar">
+      <i class="fa fa-bars" aria-hidden="true"></i>
+      <ul>
+        <li><i class="fa fa-home" aria-hidden="true"></i> Home</li>
+        <li><i class="fa fa-address-book-o" aria-hidden="true"></i>About</li>
+        <li><a href="https://twitter.com/EvitaAFD"> <i class="fa fa-twitter" aria-hidden="true"></i> Twitter</a></li>
+        <li><a href="https://github.com/EvitaAFD"><i class="fa fa-github" aria-hidden="true"></i> GitHub</a></li>
+      </ul>
+    </nav>
+    <header class="site-heading clearfix">
+      <section class="heading-group-h1">
+        <h1>Eve Denison</h1>
+      </section>
+    </header>
+    <section id="about" class="tab-content clearfix">
+      <!-- github projects will go here -->
+   <header>
+    <section>
+      <h1>About Me</h1>
```

Comment:
As good practice, you should only have a single `<h1>` element in your page, containing the page's title. Any other section headers should be `<h2>` at most.